### DISCUSSION TITLE
New predefined options

### DISCUSSION THREAD

**Discussion by nadiadavidson** [DavidsonGroup/flexiplex#28] [Show and tell]
@ChangqingW @olliecheng I've just pushed a new version which adds presets for 10x 3' version 3, 3' version 2 and 5' version 2, plus grep-like search. Please check it out and let me know if there are any issues. Once we get some Visium, ParseBio, etc. data to play with, we can add these in too.

I'm also starting a new version numbering system when we use add .1 to the end for the dev versions and then remove or increment for the release.

**Comment by ChangqingW** [comment]
flexiplex should be able to demultiplex the [SiT (Nanopore) data](https://doi.org/10.1093/nar/gkad169). @youyupei is also re-analysing that data.

**Reply by nadiadavidson** [reply]
Thanks @ChangqingW, just checked some of their reads and it looks like the barcode structure is identical to single cell 3' v3. Did you find the same @youyupei?

**Reply by youyupei** [reply]
Hi @nadiadavidson ,

I think you are right. It seems to be the  same adapter sequence, same length of barcode and UMI. I didn't see any difference, except that the 16nt barcodes are coming from a different list, which contains ~5k unique barcode and is provided in the Space Ranger. 
I think flexiplex should be directly applicable.
